content/federatedOrcfaxNetwork.ttl

```diff
@@ -1,7 +1,10 @@
 @prefix : <http://glossary.orcfax.io/#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :federatedOrcfaxNetwork
-    skos:prefLabel "federated Orcfax network"@en;
+    skos:prefLabel "Federated Orcfax network"@en;
```

Comment:
Odd capitalizition choice here

Reply:
I believe Vahid and Peter have used Federated Orcfax Network as a proper noun... however I see I forgot to capitalize "network"-- was this hat you were referencing or the capitalization of "federated"?